src/monocle_apptrace/wrap_common.py

```diff
@@ -266,3 +290,23 @@ def update_tags(instance, span):
         span.set_attribute(TAGS, [model_name, vector_store_name])
     except:
         pass
+
+
+def update_vectorstore_attributes(to_wrap, instance, span):
+    """
+       Updates the telemetry span attributes for vector store retrieval tasks.
+    """
+    try:
+        package = to_wrap.get('package')
+        if package in frameworks_mapping:
+            attributes = frameworks_mapping[package](instance)
+            span._attributes.update({
+                TYPE: attributes['type'],
+                PROVIDER: attributes['provider'],
+                EMBEDDING_MODEL: attributes['embedding_model']
+            })
+        else:
+            print(f"Package '{package}' not recognized for vector store telemetry.")
```

Comment:
use logger instead of print, below as well

Reply:
done